lib/engine/game/g_18_oe/step/buy_sell_par_shares.rb

```diff
@@ -16,23 +26,69 @@ def actions(entity)
             actions << 'par' if can_ipo_any?(entity) || can_float_minor?(entity)
             actions << 'buy_company' if !purchasable_companies(entity).empty? || !buyable_bank_owned_companies(entity).empty?
             actions << 'sell_shares' if can_sell_any?(entity)
-
-            actions << 'pass' if !can_float_minor?(entity) && !bought? && !actions.empty?
+            actions << 'convert' unless @can_convert.empty?
```

Comment:
This line isn't needed. The earlier call to `corporation_actions` will have added convert to the action list if any conversions are possible.

Reply:
I tried that and it always breaks the game, not sure why. My guess is the convert isn't getting added into the player actions list only the corporations?

Reply:
Is this breaking for new games, or loading ones that were created before the code was changed? If it's breaking for loading old games then this might mean that there is a pass action in the game that is no longer needed. I can't immediately see why this would have changed the behaviour.

Reply:
Breaks both. If convert is the only action left for a player it auto passes them as `actions.empty?` is true for the player

Reply:
OK, leave this in if it's not working without it. I didn't need to add a convert action here in 1858, which also uses `corporation_actions`, so I didn't think it would be needed for you.

[Apologies for not replying sooner, I missed that you'd added another comment].